content/updates/2017-11-20-release-notes-buildpacks-volume-services-other-new-features.md

```diff
@@ -0,0 +1,54 @@
++++
+date = "2017-11-20"
+title = "cloud.gov monthly update: Buildpack notifications, volume services, and other new features"
++++
+We’ve been hard at work shipping out new features to help you make your apps better, more secure, and improve your experience developing and deploying them. There’s a lot more in here than usual. We hope you see some features and additions in here that you’ve been waiting for. As always, get in touch with us if you have any questions or feedback about these features. See you next month.
+
+### Added
+
+* Part of keeping your app on cloud.gov secure is [using the latest version of your app’s buildpack](https://cloud.gov/docs/getting-started/app-maintenance/). In addition to keeping you updated in these release notes, we’ll now send you an email notification when a new version of your buildpack is available.
+* We have experimental support for [volume services](https://cloud.gov/docs/apps/experimental/volume-services/). Since a cloud.gov app normally has a short-lived file system, this enables you to build an app that requires persistent file storage. [Contact us if you’d like to try this](https://cloud.gov/docs/apps/experimental/volume-services/#how-to-use-this-service).
+* One of our engineers gave us the ability to make [Mermaid diagrams](https://mermaidjs.github.io/) on the cloud.gov website, and we almost got carried away creating visualizations of different parts of the platform. Here are a couple we think will help you out:
+  * A comparison of how apps with custom domains and those with the default \*.app.cloud.gov domain are served to users: https://cloud.gov/docs/apps/custom-domains/#comparison-of-default-domains-and-custom-domains.
+  * How to incorporate a CI/CD workflow into your app: https://cloud.gov/docs/apps/continuous-deployment/#configure-your-service
+* The cloud.gov dashboard now lets you manage environment variables for your apps. To see it in action, go to an app page in the dashboard and try viewing, adding, or editing an environment variable. You can also see all the environment variables set by the system from the same view.
```

Comment:
Suggest linking to https://dashboard.fr.cloud.gov/ for convenience for people who want to go click and check it out